tests/test_mons.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2020 SUSE LINUX GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import os
+import re
+import pytest
+
+logger = logging.getLogger(__name__)
+
+
+# check if rbd service gets started automatically
+def test_service_mons(rook_cluster):
+    # get configured number of monitors
+    pattern = re.compile(r'\bcount:\s+(\d+)')
+    cluster_yaml = os.path.join(rook_cluster.ceph_dir, 'cluster.yaml')
+
+    count = 0
+    for line in (open(cluster_yaml)):
+        number = pattern.findall(line)
+        for count in number:
+            count = int(count)
```

Comment:
this will take the last count found rather than specifically the count for mons.